src/main/java/dev/jbang/cli/Wrapper.java

```diff
@@ -26,9 +26,9 @@
 	@CommandLine.Command(name = "install", description = "Install/Setup jbang as a `wrapper` script in a folder")
 	public Integer install(
 			@CommandLine.Option(names = { "-d",
-					"--dir" }, description = "The folder to install the wrapper into.", defaultValue = ".") Path dest,
+					"--dir" }, description = "The folder to install the wrapper into.") Path dest,
```

Comment:
why remove default values? they show in the docs.

Reply:
Because default values are now handled by the configuration, if a user doesn't enter a value Picocli will fall back on the default value, for which it provides an interface that you can implement to supply those default values in case they are not fixed/static.

> they show in the docs.

But do they? I checked when I started implementing this and now I checked again and I don't see the default values mentioned anywhere when I execute `jbang wrapper --help` (or any other help).

I also see examples in the Picocli docs like this:

```
@Option(names = "-c", description = "The count (default: ${DEFAULT-VALUE})")
```

which seem to suggest you have to write it explicitly into the docs. This is also the recommended way to do things according to: https://picocli.info/#_show_default_values

Now there used to be a `showDefaultValues` option before Picocli version 3.0 but it's deprecated and we're not using it anywhere in our code, so perhaps it was once like this but it was lost some time ago? I don't remember seeing any default values at least.

Reply:
it works using fallback-value which is nice!